tests/Base.php

```diff
@@ -213,6 +213,15 @@ public function testRequestMethod(): void
         $response = $this->execute(method: 'PATCH', headers: ['x-action' => 'requestMethod']);
```

Comment:
Above this test is `OPTIONS`, seems to only be problematic on C++. Let's please enable it as part of this PR. Ill also take a look at all failing tests, and give helping hand. If neither of us can figure it out, we can again disable the test, but specifically for C++ only.

Reply:
Sure thing.